app/channels/notifications_channel.rb

```diff
@@ -0,0 +1,5 @@
+class NotificationsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "notifications:1"
```

Comment:
Is the `1` here, and down in the Redis URL below, a Redis "database" to keep these keys cordoned off from other ones?